components/button/button-move.js

```diff
@@ -144,16 +144,16 @@ class ButtonMove extends ThemeMixin(FocusMixin(RtlMixin(LitElement))) {
 
 				.up-layer,
 				.down-layer {
-					height: 1.1rem;
+					height: 1.2rem;
```

Comment:
This ONLY increases the height of the click targets -- the icons themselves remain `0.85rem` (`17px`) high (by `18px` wide). That's why I'm not expecting any vdiff changes here. @geurts let me know if you'd also like me to increase the height of the icons themselves by `1px` each -- that will have an impact.